.gitignore

```diff
@@ -5,4 +5,5 @@ package-lock.json
 .next
 .env.local
 yarn-error.log
-coverage
\ No newline at end of file
+coverage
+certificate
```

Comment:
can we add `.yarn` as well ?

Reply:
Can you please see this link [.yarn in .gitignore](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored)

Reply:
The .yarn file doesn't get created in the version of yarn we are using

Reply:
Currently the yarn version is not defined.
Ok got it thanks